wpeverest/everest-forms
# Fix - Date and time field option and responsive design

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes the design bug related to date and time option field and broken design in the responsive device at a minimum width of 768 pixels.

### How to test the changes in this Pull Request:

1. Add the date and time field in the form builder.
2. Click the field and on the sidebar check on the options.
3. On the Advanced Option check the check-boxes options design on the Date Format.
4. For the responsive design test, check the builder design at minimum 768 pixels.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Responsive design issue.
> Fix - Date and time field advanced option design.
